db/seeds.rb

```diff
@@ -8,14 +8,82 @@
 #     MovieGenre.find_or_create_by!(name: genre_name)
 #   end
 
-# Create the default user
-user = User.create_or_find_by!(
-  first_name: "Jeremy", last_name: "Caley", email: "jeremy",
-  password: "password", password_confirmation: "password"
-)
+require 'faker'
 
-user.workspaces.create_or_find_by!(name: "Caley", owner_id: user.id)
-puts "Workspace created: #{user.workspaces.first.name}"
+email_providers = ['gmail.com', 'yahoo.com', 'hotmail.com', 'outlook.com', 'aol.com']
 
-user.teams.create_or_find_by!(name: "Caley", workspace_id: user.workspaces.first.id)
-puts "Team created: #{user.teams.first.name}"
+# Create multiple users
+5.times do
+  name = Faker::Name.first_name
+  user = User.create!(
+    first_name: name,
+    last_name: Faker::Name.last_name,
+    email: name,
+    password: "password", 
+    password_confirmation: "password"
+  )
+
+  puts "Loading seed data for user: #{user.email}"
+
+  # Create a workspace for each user
+  workspace = Workspace.create!(
+    name: "#{Faker::Company.name} Workspace",
+    avatar_url: Faker::Company.logo,
+    owner: user
+  )
+
+  # Create a team within the workspace
+  team = Team.create!(
```

Comment:
Even if we decide to go with multiple users, it might be useful to put multiple users in a same team instead of creating unique team per user

Reply:
Agree here also